### ISSUE TITLE
Navbar not collapsing after choosing nav-item

### ISSUE THREAD

**Issue by sarru1291** [hossainemruz/toha#102]
![Screenshot from 2020-10-18 14-12-19](https://user-images.githubusercontent.com/22962336/96362653-76029700-114c-11eb-871c-7d84a8f2184d.png)

**Comment by hossainemruz** [comment]
Can you please share your device info?

**Comment by sarru1291** [comment]
Well, I've Samsung J2 but it's also not working on the browser device toolbar as well.

**Comment by alex-bezek** [comment]
I can recreate this on desktop as well. FWIW dropping
```js
      var navMain = $(".navbar-collapse");
      navMain.on("click", "a", null, function (e) {
        $('.navbar-collapse').collapse('hide');
      });
```
here https://github.com/hossainemruz/toha/blob/master/static/assets/js/navbar.js#L31 seems to resolve the issue. 
@hossainemruz  I haven't used the bootstrap menu too much, I took the answer from https://stackoverflow.com/questions/14203279/bootstrap-close-responsive-menu-on-click  
There might be a cleaner solution given this is such a common bootstrap usecase. I can keep looking

**Comment by alex-bezek** [comment]
This actually looks like a better solution
https://stackoverflow.com/questions/42401606/how-to-hide-collapsible-bootstrap-4-navbar-on-click
https://getbootstrap.com/docs/4.0/components/collapse/

I tested and adding data attributes to 
https://github.com/hossainemruz/toha/blob/master/layouts/partials/navigators/navbar.html#L44
https://github.com/hossainemruz/toha/blob/master/layouts/partials/navigators/navbar.html#L34
seems to resolve the issue in desktop chrome

**Comment by alex-bezek** [comment]
Opened a PR here https://github.com/hossainemruz/toha/pull/112

**Comment by hossainemruz** [comment]
Fixed in #112
